website/source/guides/nomad-metrics.html.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: "guides"
+page_title: "Setting up Nomad with Grafana and Prometheus Metrics"
+sidebar_current: "guides-nomad-metrics"
+description: |-
+  It is possible to collect metrics on Nomad and create dashboards with Grafana
+  and Prometheus. Nomad has default configurations for these, but it is
+  possible to build and customize these.
+---
+
+# Setting up Nomad with Grafana and Prometheus Metrics
+
+Often aggregating and displaying metrics in dashboards can lead to more useful
+insights about a cluster. It is easy to get lost in a sea of logs!
+
+This guide explains how to set up configuration for Prometheus and Grafana to
+integrate with a Nomad cluster. While this introduces the basics to get a
+dashboard up and running, Nomad exposes a wide vareity of metrics, which can be
+explored via both Grafana and Prometheus.
+
+## What are Prometheus and Grafana?
+
+Amongst other great metrics visualization tools, Prometheus and Grafana both
+aggregate data and provide an easy eay to visuqlize data.
+
+In addition to these tools, [Graphite](https://graphiteapp.org/) and
```

Comment:
I would enumerate all the supported backends. We aren't trying to show preference.